packages/rlc-common/test/integration/packageJson.spec.ts

```diff
@@ -299,7 +299,7 @@ describe("Package file generation", () => {
       );
       expect(packageFile.scripts).to.have.property(
         "build",
-        "npm run clean && dev-tool run build-package && mkdirp ./review && dev-tool run extract-api"
+        "npm run clean && dev-tool run build-package && dev-tool run vendored mkdirp ./review && dev-tool run extract-api"
```

Comment:
I think we need to apply clean script change in RLC and modular also, but I didn't notice any change in rlc-common, as well as the UT changes.